index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Adaobi</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+    <nav class="container">
+      <a href="#" class="navbar-logo">Adaobi Okoye</a>
+            </a>
+      <a href="#" class="navbar-portfolio">
+        <img src="/icons/Union.png" alt="portfolio"/>
+        <a href="#" class="navbar-about">
+        <img src="/icons/Union.png" alt="About"/>
+          <a href="#" class="navbar-contact">
+        <img src="/icons/Union.png" alt="Contact"/>
+    </nav>
+    </header>
+    <section class="header-box">
+      <section class="header-content">
+        <h1 class="header-title">Hey There. I'm Adaobi</h1>
+        <h2 class="header-subtitle">I’m a Software Developer</h2>
+        <p class="header-text-description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+                <div class="social">
+          <a
+            class="social-link"
+            href="#"
+            target="_blank"
+                      >
+            <img
+              class="social-icon"
+              src="/icons/github.png"
+              alt="Github"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="#"
+            target="_blank"
+                      >
+            <img
+              class="social-icon"
+              src="/icons/Linkedin icon.png"
+              alt="LinkedIn"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="#"
+            target="_blank"
+                     >
+            <img
+              class="social-icon"
+              src="/icons/angel list.png"
+              alt="AngelList"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="#"
+            target="_blank">
+           <img
+              class="social-icon"
+              src="/icons/twitter.png"
+              alt="Twitter"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="#"
+            target="_blank"
+                      >
+       <img class="social-icon"
+              src="/icons/medium.png"
+              alt="Medium"
+            />
+          </a>
+        </div>         

```

Comment:
- [ ] Great work on your project, you have done a great job, however, I think we will need to make a few changes on this section before you go onto the next step, Kindly change the `div` tag on line 31 to a `ul` tag instead and wrap its entire elements, each `a` tag in a `li` tag. 
- [ ] Please match your web app with the same `color, padding, margin, font-weight, font-height` to the original one.
- [ ] Consider adjusting your hamburger icon which is duplicated many times
- [ ] Kindly remove the white spaces shown below
- [ ] Actually you `h1` title is underline in blue , please `use text-decoration: none` to fix it
- [ YOUR DESIGN] 
<img width="318" alt="biz" src="https://user-images.githubusercontent.com/79058364/158888232-70a61d96-265a-4ae8-81fc-0b9131d2a757.png">

- [ FIGMA DESIGN]
<img width="294" alt="blackfug" src="https://user-images.githubusercontent.com/79058364/158888352-0df312cb-1f38-49a5-b4a7-adc0c0cb38ec.png">